Show Full Title in Pie Chart SVG

yari-dewalt opened the pull request:
## :bookmark_tabs: Summary

Aims to fix the title of a pie chart overflowing outside the bounds of the rendered SVG.

Resolves #5567

## :straight_ruler: Design Decisions

The goal was to keep the rendering the same but to account for the title length if it surpasses the length of the overall diagram. If so, it will update the total width of the diagram and offset the chart so it remains centered under the title text. It also maintains the margin on both sides.

### :clipboard: Tasks

Make sure you

- [x] :book: have read the [contribution guidelines](https://mermaid.js.org/community/contributing.html)
- [ ] :computer: have added necessary unit/e2e tests.
- [ ] :notebook: have added documentation. Make sure [`MERMAID_RELEASE_VERSION`](https://mermaid.js.org/community/contributing.html#update-documentation) is used for all new features.
- [x] :bookmark: targeted `develop` branch

sidharthv96 commented:
@yari-dewalt can you add some tests in `cypress/integration/rendering/pie.spec.ts` which covers the long title case?

yari-dewalt commented:
@sidharthv96 I went ahead and added some tests to cover the long title case, hopefully they're what you were looking for. I also added the default pieWidth in the totalWidth calculation in the renderer so there are not cut offs if there is no title / the title is not long enough

sidharthv96 commented:
Good set of tests @yari-dewalt!
Amazing work 🚀
